### PR TITLE
DIT 669 - Appeal Tab & Change Appeal Status

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/models/AppealStatus.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  type AppealStatus = Value
+  val IN_PROGRESS, ALLOWED, DISMISSED = Value
+
+  def format(status: Option[AppealStatus]): String = {
+    status match {
+      case Some(IN_PROGRESS) => "Under appeal"
+      case Some(ALLOWED) => "Appeal allowed"
+      case Some(DISMISSED) => "Appeal dismissed"
+      case None => "None"
+      case _ => ???
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/tariffclassificationfrontend/models/AppealStatus.scala]: Will this blow-up if we ever have a new appeal status?  Is this the equivalent of "unexpected value exception"?

Comment (SteveSugden) [reply app/uk/gov/hmrc/tariffclassificationfrontend/models/AppealStatus.scala]: Thought so.  Nice.

File: app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AppealCaseController.scala [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import uk.gov.hmrc.tariffclassificationfrontend.views
+import uk.gov.hmrc.tariffclassificationfrontend.views.CaseDetailPage
+
+import scala.concurrent.Future.successful
+
+@Singleton
+class AppealCaseController @Inject()(authenticatedAction: AuthenticatedAction,
+                                     override val caseService: CasesService,
+                                     override val messagesApi: MessagesApi,
+                                     override implicit val config: AppConfig) extends RenderCaseAction with I18nSupport {
```
Comment (pllaurado) [inline app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AppealCaseController.scala]: I18nSupport is already in RenderCaseAction

Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AppealCaseController.scala]: Done

**Pull request merged**